src/extension/ncwms/src/test/java/au/org/emii/geoserver/wms/LayerDescriptorTest.java

```diff
@@ -0,0 +1,25 @@
+package au.org.emii.geoserver.wms;
+
+import junit.framework.TestCase;
+import org.junit.Test;
+
+public class LayerDescriptorTest extends TestCase {
+    @Test
+    public void testSimpleParsing() throws Exception {
+        LayerDescriptor layerDescriptor;
+
+        layerDescriptor = new LayerDescriptor("imos:some_layer/sea_water_velocity");
+        assertEquals("imos", layerDescriptor.workspace);
+        assertEquals("some_layer", layerDescriptor.layer);
```

Comment:
I would still argue that `some_layer` is the "layer" in this context, it is a geoserver layer. Variable is the variable you want to extract from the referred NetCDF file.

So from the point of view of geoserver (which is this current point of view), the layer is the `_url` abomination, and variable is `sea_water_velocity` in the referred file.

From the thredds point of view, `sea_water_velocity` is the "layer" that you want from the file, which has the same name as the NetCDF variable you're after.